### PR TITLE
Allow to add additional binds to Docker build container

### PR DESCRIPTION
**Pull request by fabianmp**
Sometimes it might be necessary to add additional binds to the build
container, e.g. to add SSH keys for the docs user to clone private
repositories or for caches. These additional binds can be specified
as field in settings:

DOCKER_ADDITIONAL_BINDS = {
  '/home/docs/.ssh': {
    'bind': '/home/docs/.ssh',
    'mode': 'ro',
  }
}

### CODE REVIEW

**Review by stsewd — COMMENTED** [review]
Thanks for the PR! We are prefixing new settings with `RTD`, so this should be `RTD_DOCKER_ADDITIONAL_BINDS`

**Comment by fabianmp** [comment]
thanks for the review, I have updated the commit with this change

File: readthedocs/doc_builder/environments.py [hunk 1]
```diff
@@ -991,6 +991,8 @@ def _get_binds(self):
                 },
             }
 
+        binds.update(getattr(settings, 'RTD_DOCKER_ADDITIONAL_BINDS', {}))
```
Comment (stsewd) [inline readthedocs/doc_builder/environments.py:994]:
```suggestion
        binds.update(settings.RTD_DOCKER_ADDITIONAL_BINDS)
```

File: readthedocs/settings/base.py [hunk 2]
```diff
@@ -488,6 +488,8 @@ def TEMPLATES(self):
         'readthedocs/build:latest': DOCKER_IMAGE_SETTINGS.get('readthedocs/build:6.0'),
         'readthedocs/build:testing': DOCKER_IMAGE_SETTINGS.get('readthedocs/build:7.0'),
     })
+    # additional binds for the build container
```
Comment (stsewd) [inline readthedocs/settings/base.py:491]:
```suggestion
    # Additional binds for the build container
```

**Review by stsewd — APPROVED** [review]
Thanks

**Pull request merged**